lib/sparse/src/index/search_context.rs

```diff
@@ -51,12 +52,20 @@ impl<'a> SearchContext<'a> {
         // The max contribution per posting list that we calculate is not made to compute the max value of two negative numbers.
         // This is a limitation of the current pruning implementation.
         let use_pruning = query.values.iter().all(|v| *v >= 0.0);
+        // find min record id across all posting lists
+        let mut min_record_ids = FixedLengthPriorityQueue::new(query.indices.len());
```

Comment:
I'm not sure right now about safe capacity value. It seems we can use just `usize::MAX`, isn't it?

Reply:
We can't put a very large number as capacity because the `BinaryHeap` actually pre-allocates it.

Also, by construction there will be at most `query.indices.len()` element due a careful dance of peek/pop/push.

Do you have a specific concern in mind?

Reply:
I forgot about preallocation, nevermind